### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by avo2003**

### CODE REVIEW

File: src/users/repository/userRepository.ts [hunk 1]
```diff
@@ -49,11 +49,11 @@ export class UserRepository {
   }
 
   async changeSalary(id: number, salary: number): Promise<User> {
-    const changeSal = await this.userRepository.preload({
+    const Salary = await this.userRepository.preload({
       id: id,
       salary: salary
```
Comment (ArthurO2002) [inline src/users/repository/userRepository.ts]: variable name should be with lowercase

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]
change PR title, do propper commit with propper error message 

like: FIX: resolve comments under pull request

**Review by ArthurO2002 — DISMISSED** [review]

File: src/users/repository/userRepository.ts [hunk 2]
```diff
@@ -48,12 +48,12 @@ export class UserRepository {
...
-  async changeSalary(id: number, salary: number): Promise<User> {
-    const changeSal = await this.userRepository.preload({
+  async changeSalary(id: number, salaryAmount: number): Promise<User> {
+    const salary = await this.userRepository.preload({
       id: id,
-      salary: salary
+      salary: salaryAmount
     });
-    return this.userRepository.save(changeSal);
+    return this.userRepository.save(salary);
```
Comment (harutyunyan611) [inline src/users/repository/userRepository.ts]: it's not salary I think

**Review by harutyunyan611 — CHANGES_REQUESTED** [review]

**Review by harutyunyan611 — APPROVED** [review]
LGTM

**Pull request merged**